jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```
@@ -16,7 +16,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
     },
 
     versions+:: {
-      kubeStateMetrics: 'v1.6.0',
+      kubeStateMetrics: 'v1.7.0-rc.1',
```

```suggestion
      kubeStateMetrics: 'v1.7.1',
```

We should use that, as it includes some bug fixes https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.7.1